processing file asmFiles/branch.asm (1 of 20)
processing file asmFiles/count_days.asm (2 of 20)
processing file asmFiles/fib.asm (3 of 20)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
--- memsim.hex	2021-10-07 13:30:49.313388235 -0400
+++ memcpu.hex	2021-10-07 13:30:49.246385697 -0400
@@ -1,6 +1,6 @@
-:04000000340100F0D7
-:0400010034020100C4
-:0400020034030200C1
+:040000000000733752
+:0400010000002701D3
+:0400020000001337B0
 :0400030034040300BE
 :0400040034050400BB
 :040005008C26000045
@@ -15,7 +15,4 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
 :00000001FF
processing file asmFiles/test.ldst2.asm (3 of 5)
--- memsim.hex	2021-10-07 13:30:50.676439857 -0400
+++ memcpu.hex	2021-10-07 13:30:50.599436940 -0400
@@ -1,6 +1,5 @@
-:040000003C0AFEEDCB
-:0400010034010F00B7
-:0400020034020800BC
+:040000000000733752
+:0400020000001337B0
 :04000300354ABEEFCD
 :040004008C23000049
 :040005008C24000443
@@ -10,10 +9,7 @@
 :04000900AC450008FA
 :04000A00AC4A000CF0
 :04000B00FFFFFFFFF5
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
+:04020300FEED00000C
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2021-10-07 13:30:52.121494584 -0400
+++ memcpu.hex	2021-10-07 13:30:52.062492349 -0400
@@ -1,6 +1,5 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
+:040000000000733752
+:0400020000001337B0
 :0400030034E7BEEF31
 :040004008C23000049
 :040005008C24000443
@@ -10,10 +9,7 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
+:04002300DEAD00004E
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/branch.asm (1 of 20)
processing file asmFiles/count_days.asm (2 of 20)
processing file asmFiles/fib.asm (3 of 20)
processing file asmFiles/halt.asm (4 of 20)
processing file asmFiles/hazard.control.asm (5 of 20)
processing file asmFiles/hazard.data.asm (6 of 20)
processing file asmFiles/hazard.structure.asm (7 of 20)
processing file asmFiles/jump.asm (8 of 20)
processing file asmFiles/l7_fib.asm (9 of 20)
processing file asmFiles/l7_mult.asm (10 of 20)
processing file asmFiles/l7_search.asm (11 of 20)
processing file asmFiles/mergesort.asm (12 of 20)
--- memsim.hex	2021-10-07 13:38:16.543325872 -0400
+++ memcpu.hex	2021-10-07 13:38:16.428321517 -0400
@@ -80,134 +80,74 @@
 :04004F0010E00001BC
 :040050000800004A5A
 :0400510003E00008C0
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000438
+:0400C1000000000338
+:0400C2000000000A30
+:0400C3000000001524
+:0400C4000000001523
+:0400C5000000001621
+:0400C6000000001620
+:0400C7000000001B1A
+:0400C8000000001B19
+:0400C9000000002013
+:0400CA000000002012
+:0400CB0000000038F9
+:0400CC0000000038F8
+:0400CD000000003DF2
+:0400CE000000003DF1
+:0400CF000000003DF0
+:0400D0000000003DEF
+:0400D10000000042E9
+:0400D20000000042E8
+:0400D3000000004BDE
+:0400D4000000004BDD
+:0400D5000000004DDA
+:0400D6000000004DD9
+:0400D70000000050D5
+:0400D80000000050D4
+:0400D90000000051D2
+:0400DA0000000051D1
+:0400DB0000000053CE
+:0400DC0000000053CD
+:0400DD000000005AC5
 :0400DE000000005AC4
 :0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
+:0400E00000000063B9
+:0400E1000000000813
+:0400E2000000000812
+:0400E3000000001108
+:0400E4000000001107
+:0400E5000000001AFD
+:0400E6000000001AFC
+:0400E70000000020F5
+:0400E80000000020F4
+:0400E90000000020F3
+:0400EA0000000020F2
+:0400EB0000000026EB
+:0400EC0000000026EA
+:0400ED0000000026E9
+:0400EE0000000026E8
+:0400EF0000000028E5
+:0400F00000000028E4
+:0400F10000000032D9
+:0400F20000000032D8
+:0400F30000000041C8
+:0400F40000000041C7
+:0400F50000000042C5
+:0400F60000000042C4
+:0400F70000000044C1
+:0400F80000000044C0
+:0400F9000000005BA8
+:0400FA000000005BA7
+:0400FB000000005CA5
+:0400FC000000005CA4
+:0400FD000000005CA3
 :0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0400FF000000006499
+:0401000000000043B8
+:0401400000000003B8
+:040142000000000AAF
+:0401440000000015A2
+:0401450000000015A1
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (13 of 20)
processing file asmFiles/mult_procedure.asm (14 of 20)
processing file asmFiles/search.asm (15 of 20)
processing file asmFiles/test.branchjump1.asm (16 of 20)
processing file asmFiles/test.ldst1.asm (17 of 20)
processing file asmFiles/test.ldst2.asm (18 of 20)
processing file asmFiles/test.loadstore.asm (19 of 20)
processing file asmFiles/test.rtype.asm (20 of 20)
processing file asmFiles/branch.asm (1 of 20)
processing file asmFiles/count_days.asm (2 of 20)
processing file asmFiles/fib.asm (3 of 20)
processing file asmFiles/halt.asm (4 of 20)
processing file asmFiles/hazard.control.asm (5 of 20)
processing file asmFiles/hazard.data.asm (6 of 20)
processing file asmFiles/hazard.structure.asm (7 of 20)
processing file asmFiles/jump.asm (8 of 20)
processing file asmFiles/l7_fib.asm (9 of 20)
processing file asmFiles/l7_mult.asm (10 of 20)
processing file asmFiles/l7_search.asm (11 of 20)
processing file asmFiles/mergesort.asm (12 of 20)
processing file asmFiles/mult.asm (13 of 20)
processing file asmFiles/mult_procedure.asm (14 of 20)
processing file asmFiles/search.asm (15 of 20)
processing file asmFiles/test.branchjump1.asm (16 of 20)
processing file asmFiles/test.ldst1.asm (17 of 20)
processing file asmFiles/test.ldst2.asm (18 of 20)
processing file asmFiles/test.loadstore.asm (19 of 20)
processing file asmFiles/test.rtype.asm (20 of 20)
